test/ParseableInterface/ModuleCache/SDKDependencies.swift

```diff
@@ -49,23 +71,33 @@
 // EXLIB-DAG: /my-sdk/ExportedLib.swiftinterface
 // SDKLIB-DAG: /my-sdk/SdkLib.swiftinterface
 //
+// Check they don't contain any dependencies from either cache other than themselves
+// RUN: cat %t/MCP/ExportedLib-*.swiftmodule | %FileCheck %s -check-prefix=NOCACHE -DLIB_NAME=ExportedLib
+// RUN: cat %t/MCP/SdkLib-*.swiftmodule | %FileCheck %s -check-prefix=NOCACHE -DLIB_NAME=SdkLib
+//
+// NOCACHE: dependencies:
+// NOCACHE: /prebuilt-cache/[[LIB_NAME]].swiftmodule
+// NOCACHE-NOT: /prebuilt-cache/
+// NOCACHE-NOT: /MCP/
```

Comment:
The way negative lit checks work is that they can't appear between two positive checks, or between the last positive check and the end of the file. So this doesn't check that there aren't more dependencies *before* the one you've named.